plugins/woocommerce-blocks/tests/e2e/utils/create-dynamic-content.ts

```diff
@@ -43,7 +43,13 @@ const createPost = async (
 	requestUtils: RequestUtils,
 	payload: CreatePostPayload
 ) => {
-	const post = await requestUtils.createPost( payload );
+	// The underlying createPost method passes the payload as URI params, triggering URI too long errors
+	// if you pass long blog post content.
+	const post = await requestUtils.rest( {
+		method: 'POST',
+		path: `/wp/v2/posts`,
+		data: { ...payload },
+	} );
```

Comment:
This is interesting. Should we report it upstream?

Reply:
@WunderBart yes as I mentioned in my PR description I plan to 🙇🏻

Reply:
@WunderBart raised here: https://github.com/WordPress/gutenberg/issues/59427